linkSampleApp/src/main/java/org/kinecosystem/linkSampleApp/LinkMainActivity.java

```diff
@@ -4,36 +4,33 @@
 import android.content.Intent;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
-import android.util.Log;
-import android.widget.Button;
+import android.text.TextUtils;
 
 import org.kinecosystem.baseSampleApp.SampleBaseActivity;
 import org.kinecosystem.baseSampleApp.SampleBaseApplication;
 import org.kinecosystem.baseSampleApp.sampleWallet.SampleWallet;
-import org.kinecosystem.linkwallet.LinkingClient;
+import org.kinecosystem.onewallet.IOneWalletClient;
+import org.kinecosystem.onewallet.OneWalletClient;
+
+import kin.sdk.KinAccount;
 
 public class LinkMainActivity extends SampleBaseActivity {
 
-    private static final int REQUEST_CODE = 50;
+    private static final int ONE_WALLET_REQUEST_CODE = 50;
 
-    private LinkingClient linkingClient;
+    private IOneWalletClient oneWalletClient;
```

Comment:
don't see a reason for having an interface for `OneWallerClient`

Reply:
no real reason - just making clearer what the API is